pkg/util/config.go

```diff
@@ -22,6 +22,8 @@ import (
 	"reflect"
 	"strconv"
 	"strings"
+
+	utilnet "k8s.io/kubernetes/pkg/util/net"
```

Comment:
Should we be using the client-go library for structs like this?

"k8s.io/client-go/1.5/pkg/util/net"

https://github.com/kubernetes/client-go/blob/c72e2838b9cfac95603049d57c9abba12e587fff/pkg/util/net/port_range.go